### PR TITLE
Docker-Compose no more

### PR DESCRIPTION
**Pull request by mnaamani**
Update all scripts, docs and automation to not install or use `docker-compose`, replaced with `docker compose` command. It is no longer recommended, and it is not found on the github runners so some github workflows are failing.

### CODE REVIEW

File: devops/ansible/roles/common/tasks/install-tools.yml [hunk 1]
```diff
@@ -74,16 +73,16 @@
...
-  become: true
-  shell:
-    cmd: |
-      bash -c "
-        curl -L 'https://github.com/docker/compose/releases/download/v2.20.1/docker-compose-$(uname -s)-$(uname -m)' -o /usr/local/bin/docker-compose
-        chmod +x /usr/local/bin/docker-compose
-        ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
-      "
-    creates: /usr/local/bin/docker-compose
+# - name: Install docker-compose
```
Comment (freakstatic) [inline devops/ansible/roles/common/tasks/install-tools.yml]: this part can probably be removed

File: devops/ansible/roles/common/tasks/install-tools.yml [hunk 2]
```diff
@@ -43,7 +43,6 @@
       - docker-compose
```
Comment (freakstatic) [inline devops/ansible/roles/common/tasks/install-tools.yml:43]: I think this line is no longer needed

**Comment by freakstatic** [comment]
@mnaamani you missed the README.md of the QN 
https://github.com/mnaamani/joystream/blob/bee4f6b6b6d3c1204ac11f432f24125e12a33478/query-node/README.md#L137

**Review by freakstatic — CHANGES_REQUESTED** [review]

File: devops/ansible/roles/common/tasks/install-tools.yml (hunk 2 above)
Comment (mnaamani) [reply devops/ansible/roles/common/tasks/install-tools.yml:43]: This Ansible task will actually uninstall docker-compose. See the later line `state: absent` so the `apt` module will remove it if found.

File: devops/ansible/roles/common/tasks/install-tools.yml (hunk 1 above)
Comment (mnaamani) [reply devops/ansible/roles/common/tasks/install-tools.yml]: Agreed

File: devops/ansible/roles/common/tasks/install-tools.yml (hunk 2 above)
Comment (freakstatic) [reply devops/ansible/roles/common/tasks/install-tools.yml:43]: Oh I see, makes sense then 👍

**Review by freakstatic — APPROVED** [review]
LGTM!

**Review by freakstatic — APPROVED** [review]

**Pull request merged**